Library/SortParams.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recurly
+{
+    public class SortParams
```

Comment:
I don't think this is the best name for the class. And maybe it should be namespaced somewhere else.